### PR TITLE
Update Validation.english.php

### PR DESCRIPTION
**Pull request by live627**
make some sense of the error messages; some now more accurately reflect the methods.

### CODE REVIEW

File: themes/default/languages/english/Validation.english.php [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
-$txt['_validate_limits'] = 'The %1$s field contains a value outside of allowable limits %2$s';
-$txt['_validate_generic'] = 'The %1$s field contains an invalid value';
-$txt['validation_failure'] = 'The form has the following error or errors that must be corrected before continuing:';
\ No newline at end of file
+$txt['_validate_required'] = 'The %1$s field is required.';
+$txt['_validate_valid_email'] = 'The %1$s field is required to be a valid email address.';
+$txt['_validate_max_length'] = 'The %1$s field can hold a maximum of %2$s characters.';
+$txt['_validate_min_length'] = 'The %1$s field needs to be a minimun of %2$s characters.';
+$txt['_validate_length'] = 'The %1$s field needs to be exactly %2$s character in length.';
+$txt['_validate_alpha'] = 'The %1$s field can only contain alpha characters (a-z).';
```
Comment (Frenzie) [inline themes/default/languages/english/Validation.english.php]: If [a-z] is indeed all it takes it'd be better to say something like `lower case letters` or possibly `lower case letters without diacritics`.

File: themes/default/languages/english/Validation.english.php [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
+$txt['_validate_alpha_numeric'] = 'The %1$s field can only contain alpha-numeric characters.';
+$txt['_validate_alpha_dash'] = 'The %1$s field can only contain alpha characters &amp; dashes.';
+$txt['_validate_numeric'] = 'The %1$s field should be numeric.';
+$txt['_validate_integer'] = 'The %1$s field can only contain an integer.';
+$txt['_validate_boolean'] = 'The %1$s field can only contain a boolean.';
+$txt['_validate_float'] = 'The %1$s field can only contain a floating-point number.';
```
Comment (Frenzie) [inline themes/default/languages/english/Validation.english.php]: Needs an example like `(1.0)` or something.

File: themes/default/languages/english/Validation.english.php [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
+$txt['_validate_valid_email'] = 'The %1$s field is required to be a valid email address.';
+$txt['_validate_max_length'] = 'The %1$s field can hold a maximum of %2$s characters.';
+$txt['_validate_min_length'] = 'The %1$s field needs to be a minimun of %2$s characters.';
+$txt['_validate_length'] = 'The %1$s field needs to be exactly %2$s character in length.';
+$txt['_validate_alpha'] = 'The %1$s field can only contain alpha characters (a-z).';
+$txt['_validate_alpha_numeric'] = 'The %1$s field can only contain alpha-numeric characters.';
+$txt['_validate_alpha_dash'] = 'The %1$s field can only contain alpha characters &amp; dashes.';
+$txt['_validate_numeric'] = 'The %1$s field should be numeric.';
+$txt['_validate_integer'] = 'The %1$s field can only contain an integer.';
+$txt['_validate_boolean'] = 'The %1$s field can only contain a boolean.';
```
Comment (Frenzie) [inline themes/default/languages/english/Validation.english.php]: I.e., `true or false`. Who's going to know what a boolean is?

File: themes/default/languages/english/Validation.english.php [hunk 4]
```diff
@@ -1,28 +1,28 @@
...
+$txt['_validate_valid_ipv6'] = 'The %1$s field needs to contain a valid IPv6 address.';
+$txt['_validate_contains'] = 'The %1$s field needs to contain one of these values: %2$s.';
+$txt['_validate_invalid_function'] = 'The specified validation function %1$s does not exist.';
+$txt['_validate_without'] = 'The %1$s field can not contain the %2$s character.';
+$txt['_validate_notequal'] = 'The %1$s field does not contain a valid value.';
+$txt['_validate_isarray'] = 'The %1$s field does not contain a valid array.';
+$txt['_validate_php_syntax'] = 'PHP syntax error: %2$s.';
+$txt['_validate_limits'] = 'The %1$s field contains a value outside of allowable limits %2$s.';
+$txt['_validate_generic'] = 'The %1$s field contains an invalid value.';
+$txt['validation_failure'] = 'The form has the following error or errors that must be corrected before continuing:.';
```
Comment (Frenzie) [inline themes/default/languages/english/Validation.english.php]: There should be no period at the end here.

File: themes/default/languages/english/Validation.english.php (hunk 1 above)
Comment (live627) [inline themes/default/languages/english/Validation.english.php]: the code says

> // A character with the Unicode property of letter (any kind of letter from any language)

so, what should I write?

Comment (Frenzie) [inline themes/default/languages/english/Validation.english.php]: https://github.com/elkarte/Elkarte/blob/bc0d8a640e9cc51bdc0e8c19a6d75e3c0cb5a414/sources/subs/DataValidator.class.php#L862

Right, I see. Docs here: http://php.net/manual/en/regexp.reference.unicode.php

Just `letter`?

**Comment by Spuds** [comment]
Yah, those were a bit rough around the edges .. Thanks!

**Comment by live627** [comment]
np! Happy to help!

**Pull request merged**